cheroot/ssl/builtin.py

```diff
@@ -43,20 +43,25 @@ class BuiltinSSLAdapter(Adapter):
     """The ssl.SSLContext that will be used to wrap sockets where available
     (on Python > 2.7.9 / 3.3)
     """
+    ciphers = None
+    """The ciphers list of SSL."""
```

Comment:
At least this is supported by linters. Perhaps @jaraco could link smth relevant.